proto/src/util.rs

```diff
@@ -75,8 +82,9 @@ impl TryFrom<Status> for grpcio::RpcStatus {
     type Error = grpcio::Error;
 
     fn try_from(value: Status) -> grpcio::Result<Self> {
-        #[cfg(feature = "protobuf-codec")]
-        let details = protobuf::Message::write_to_bytes(&value)?;
+        #[cfg(any(feature = "protobuf-codec", feature = "protobufv3-codec"))]
+        let details = protobuf::Message::write_to_bytes(&value)
+            .map_err(|_err| grpcio::Error::RemoteStopped)?;
```

Comment:
Why change it to `RemoteStopped`?

Reply:
This was to fix a type error, but this doesn't seem needed. I removed both occurrences.